packages/loader/src/ShaderChunkLoader.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  AssetPromise,
+  AssetType,
+  LoadItem,
+  Loader,
+  ResourceManager,
+  Shader,
+  ShaderFactory,
+  resourceLoader
+} from "@galacean/engine-core";
+
+@resourceLoader("ShaderChunk", ["glsl"], false)
+class ShaderChunkLoader extends Loader<void> {
+  load(item: LoadItem, resourceManager: ResourceManager): AssetPromise<void> {
+    return this.request<any>(item.url, { ...item, type: "text" }).then(async (code) => {
+      const { includeKey } = item.params;
+      ShaderFactory.registerInclude(includeKey, code);
```

Comment:
You registerInclude with relative path will cause another Shader include same chunk can't find register

Reply:
Finally! The `includeKey` in params is an absolute path ：）

Reply:
done